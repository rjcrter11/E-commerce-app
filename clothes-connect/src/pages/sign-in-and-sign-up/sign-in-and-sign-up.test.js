import React from 'react';
import { create } from 'react-test-renderer';

import SignInAndSignUpPage from './sign-in-and-sign-up.component';

it('should render SignIn/SignUpPage component', () => {
    const wrapper = create(<SignInAndSignUpPage />);
    expect(wrapper.toJSON()).toMatchSnapshot();
})