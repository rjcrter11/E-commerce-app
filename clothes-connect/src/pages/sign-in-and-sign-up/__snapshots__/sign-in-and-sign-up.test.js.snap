// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SignIn/SignUpPage component 1`] = `
<div
  className="sign-in-and-sign-up"
>
  <div
    className="sign-in"
  >
    <h2>
      I already have an account
    </h2>
    <span>
      Sign in with your email and password
    </span>
    <form
      onSubmit={[Function]}
    >
      <div
        className="group"
      >
        <input
          className="form-input"
          data-testid="form-input"
          name="email"
          onChange={[Function]}
          required={true}
          type="email"
          value=""
        />
        <label
          className=" form-input-label"
          data-testid="input-label"
        >
          email
        </label>
      </div>
      <div
        className="group"
      >
        <input
          className="form-input"
          data-testid="form-input"
          name="password"
          onChange={[Function]}
          required={true}
          type="password"
          value=""
        />
        <label
          className=" form-input-label"
          data-testid="input-label"
        >
          password
        </label>
      </div>
      <div
        className="buttons"
      >
        <button
          className="   custom-button"
          type="submit"
        >
          SIGN IN
        </button>
        <button
          className="  google-sign-in custom-button"
          onClick={[Function]}
          type="button"
        >
          Sign in with Google
        </button>
      </div>
    </form>
  </div>
  <div
    className="sign-up"
  >
    <h2
      className="title"
    >
      I do not have an account
    </h2>
    <span>
      Sign up with your email and password
    </span>
    <form
      className="sign-up-form"
      onSubmit={[Function]}
    >
      <div
        className="group"
      >
        <input
          className="form-input"
          data-testid="form-input"
          name="displayName"
          onChange={[Function]}
          required={true}
          type="text"
          value=""
        />
        <label
          className=" form-input-label"
          data-testid="input-label"
        >
          Display Name
        </label>
      </div>
      <div
        className="group"
      >
        <input
          className="form-input"
          data-testid="form-input"
          name="email"
          onChange={[Function]}
          required={true}
          type="email"
          value=""
        />
        <label
          className=" form-input-label"
          data-testid="input-label"
        >
          Email
        </label>
      </div>
      <div
        className="group"
      >
        <input
          className="form-input"
          data-testid="form-input"
          name="password"
          onChange={[Function]}
          required={true}
          type="password"
          value=""
        />
        <label
          className=" form-input-label"
          data-testid="input-label"
        >
          Password
        </label>
      </div>
      <div
        className="group"
      >
        <input
          className="form-input"
          data-testid="form-input"
          name="confirmPassword"
          onChange={[Function]}
          required={true}
          type="password"
          value=""
        />
        <label
          className=" form-input-label"
          data-testid="input-label"
        >
          Confirm Password
        </label>
      </div>
      <button
        className="   custom-button"
        type="submit"
      >
        SIGN UP
      </button>
    </form>
  </div>
</div>
`;
